lib/cli-engine.js

```diff
@@ -256,6 +256,27 @@ function isErrorMessage(message) {
     return message.severity === 2;
 }
 
+
+/**
+ * Checks if a provided path is a directory and returns a glob string matching
+ * all files under that directory if so, the path itself otherwise.
+ * Also makes sure all path separators are POSIX style for `glob` compatibility.
+ *
+ * @param {string} pathname The directory path to be modified
+ * @returns {string} The glob path or the file path itself
+ */
+function processPath(pathname) {
```

Comment:
Globbing `foo` will get the directory foo, not all its files. The glob pattern to get the files in a directory would be `foo/**`. This is here to preserve the previous behaviour of `foo` getting all the files in the directory. Nothing to do with `globby` though, this is how `node-glob` works and globbing in general.